sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/models/application-offering-change-request.dto.ts

```diff
@@ -178,11 +178,18 @@ export class ApplicationOfferingChangeAssessmentAPIInDTO {
   @IsNotEmpty()
   @MaxLength(NOTE_DESCRIPTION_MAX_LENGTH)
   note: string;
-  @IsIn([
-    ApplicationOfferingChangeRequestStatus.Approved,
-    ApplicationOfferingChangeRequestStatus.DeclinedBySABC,
-  ])
-  applicationOfferingChangeRequestStatus: ApplicationOfferingChangeRequestStatus;
+  @IsIn(
+    [
+      ApplicationOfferingChangeRequestStatus.Approved,
+      ApplicationOfferingChangeRequestStatus.DeclinedBySABC,
+    ],
+    {
+      message: `Application offering change request must be ${ApplicationOfferingChangeRequestStatus.Approved} or ${ApplicationOfferingChangeRequestStatus.DeclinedBySABC}`,
```

Comment:
The API will return an out-of-box self-explanatory message similar to this one. Is there a need to use a custom one for this scenario?

Reply:
We are using our custom validation message in a few other places in the code. I thought it would be good to keep here as well. If you want, I can remove it.

Reply:
Removed it.